ideascube/serveradmin/catalog.py

```diff
@@ -35,6 +35,32 @@ def rm(path):
         shutil.rmtree(path)
 
 
+def load_from_file(path):
+    """Load catalog data from a local file
```

Comment:
Honnestly, I think those docstrings are useless: function name and argument names should be self explanatory (and they are in this given case).
Those are usefeful only when we want to expose an API documentation, which is not the case here.